ubuntu/gpu/rapids/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM databricksruntime/gpu-python:cuda10.1
+
+COPY rapids-spec.txt /tmp/rapids-spec.txt
+
+RUN conda create --name rapids --file /tmp/rapids-spec.txt && \
+    rm -f /tmp/rapids-spec.txt
+
+# Set an environment variable used by Databricks to decide which conda environment to activate by default.
+ENV DEFAULT_DATABRICKS_ROOT_CONDA_ENV=rapids
```

Comment:
This still leaves the old `gpu` conda environment around, right? Should we remove it to save on image size? Probably not worth it for now.

Reply:
Does it really save the image size? The layers are still there. I don't expect users use those containers directly. Those are just examples.